tests/conftest.py

```diff
@@ -94,13 +94,25 @@ def app_id():
 
 
 @pytest.fixture
-def portal_mock(dbus_test_case, portal_name, required_templates, template_params, app_id) -> PortalMock:
+def portal_mock(
+    dbus_test_case,
+    portal_name,
+    required_templates,
+    template_params,
+    app_id,
+) -> PortalMock:
     """
     Fixture yielding a PortalMock object with the impl started, if applicable.
     """
-    pmock = PortalMock(dbus_test_case, portal_name, app_id)
+    pmock = PortalMock(
+        dbus_test_case,
+        portal_name,
+        app_id=app_id,
```

Comment:
Any reason you're passing the app_id by name but not the others?

Reply:
No, passing everything by position now.